tools/rcverify.sh

```diff
@@ -183,6 +177,21 @@ function analyzeKeyImport() {
     fi
 }
 
+function validateNotice() {
```

Comment:
I have tested all `NOTICE.txt` files in all repos, and confirmed it works as expected.
But it would be great if anyone cross-checks again.

Some of them require changes so I opened a couple of corresponding PRs.

Reply:
Nice!  Thank you @style95